src/vs/workbench/parts/preferences/browser/preferencesEditor.ts

```diff
@@ -164,7 +164,10 @@ export class PreferencesEditor extends BaseEditor {
 		this.searchWidget.focus();
 	}
 
-	public focusSearch(): void {
+	public focusSearch(filter?: string): void {
+		if (filter !== null) {
```

Comment:
Thanks! I wrote my first bit of TS code in this PR. I'll check out the fix to learn :)